web/client/themes/default/less/modal.less

```diff
@@ -189,7 +189,9 @@
                     padding: 9px;
                 }
             }
-
+            & > div.chart-color-class-modal-body {
+                scrollbar-gutter: stable;
+            }
```

Comment:
@allyoucanmap as I mentioned, I am not sure that this is a good fix. The problem we have here is that the scrollbar keeps 'blinking' when the modal window height is near that height that the browser calculates to show or not the scrollbar. See here the original issue.
https://github.com/geosolutions-it/MapStore2/issues/7734#issuecomment-1020014284 (see gif n.2)
Maybe you can give me some advice on how to find a better solution.

Reply:
This should be already be fixed. Could you please update and remove?